### ISSUE TITLE
Make a default for skip  (if non defined don't skip)

### ISSUE THREAD

**Issue by cinek810** [cinek810/ansible-deployer#51]
```
Traceback (most recent call last):
  File "/usr/local/ansible/bin/ansible-deployer", line 8, in <module>
    sys.exit(main())
  File "/usr/local/ansible/lib64/python3.6/site-packages/ansible_deployer/command_line.py", line 551, in main
    run_task(config, options, inv_file)
  File "/usr/local/ansible/lib64/python3.6/site-packages/ansible_deployer/command_line.py", line 388, in run_task
    playbooks = get_playbooks(config, options)
  File "/usr/local/ansible/lib64/python3.6/site-packages/ansible_deployer/command_line.py", line 370, in get_playbooks
    if item["skip"]:
KeyError: 'skip'
```

**Comment by LegenJCdary** [comment]
This is fixed in #50

**Comment by cinek810** [comment]
Well it's not invalid, it;s just fixed :)
